src/components/app-bar/app-bar.atoms.tsx

```diff
@@ -0,0 +1,19 @@
+import type { HTMLAttributes } from 'react'
+import {
+  ElNavSearchButton,
+  ElNavSearchButtonIcon,
+  ElNavSearchButtonPlaceholder,
+  ElNavSearchButtonShortcutText,
+} from './styles'
+
+export interface NavSearchButton extends HTMLAttributes<HTMLButtonElement> {}
+
+export const NavSearchButton: React.FC<NavSearchButton> = (props) => {
+  return (
+    <ElNavSearchButton {...props}>
+      <ElNavSearchButtonIcon icon="search" />
+      <ElNavSearchButtonPlaceholder>Search</ElNavSearchButtonPlaceholder>
+      <ElNavSearchButtonShortcutText>Ctrl + K</ElNavSearchButtonShortcutText>
```

Comment:
**issue:** I don't think the shortcut should not be hardcoded here. Not all products will necessarily implement the keyboard shortcut, and even if they do, some may not be able to use Ctrl + K specifically.

Reply:
hmm I haven't seen that in the design, because every example in the design have this shortcut so I thought its really required 🤔 , but I think it make sense to make it optional to be save